modules/complete-assisted-installer-oci.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_oci/installing-oci-assisted-installer.adoc [Using the Assisted Installer to install a cluster on OCI]
+
+:_mod-docs-content-type: PROCEDURE
+[id="complete-assisted-installer-oci_{context}"]
+= Completing the remaining Assisted Installer steps
+
+After you provision {oci-first} resources and upload {product-title} custom manifest configuration files to {oci}, you must complete the remaining cluster installation steps on the Assisted Installer before you can create an instance {oci}.
+
+.Prerequisites
+
+* You created a resource stack on {oci}, and the stack includes the custom manifest configuration files and {oci} Resource Manager configuration resources.
+
+.Procedure
+
+. From the link:https://console.redhat.com/[Red Hat Hybrid Cloud Console] web console, go to the **Host discovery** step. Under the **Role** column, assign a node role, `Control plane node` or `Worker`, for each targeted hostname.
++
+[IMPORTANT]
+====
+Before, you can continue to the next steps, wait for each node to reach the `Ready` status.
+====
+
+. Accept the default settings for the **Storage** and **Networking** steps. Click the **Next** button to go to the **Custom manifests** step. 
+
+. Select the value `manifests` in the **Folder** field and enter a value in the **File name** field, such as `oci-ccm.yml`. From the **Content** section, click on **Browse** and select the CCM manifest from your drive located in `custom_ manifest/manifests/oci-ccm.yml`. 
+
+. Expand the next **Custom manifest** section and repeat the same steps for the following manifests:
+    - CSI driver manifest: `custom_ manifest/manifests/oci-csi.yml`
+    - CCM machine configuration: `custom_ manifest/openshift/machineconfig-ccm.yml`
+    - CSI driver machine configuration: `custom_ manifest/openshift/machineconfig-csi.yml`
+
+. Complete the **Review and create** step to create your {product-title} cluster on {oci}. Click the **Install cluster** button to finalize the cluster installation. 
```

Comment:
```suggestion
. Complete the **Review and create** step to create your {product-title} cluster on {oci}. Click the **Install cluster** button to finish the cluster installation. 
```

Reply:
Style guide recommends "finish" vs. "finalize". I'm not 100% committed to this but thought I would pass it on. :-/